src/App.js

```diff
@@ -76,4 +72,21 @@ class App extends React.Component {
   }
 }
 
+class App extends React.Component {
+  render() {
+    return (
+      <>
+        <Router>
+          <Switch>
+          <Route exact path="/">
+            {LOGGEDIN ? <Redirect to="/experiment" /> : <Login/>}
+          </Route>
```

Comment:
route feels like a heavy construct for this? what about just an if around whether or not logged in?

Reply:
Totally agree, I think it should just be a switch between `<Login/>` and `<ExpStart/>`